src/components/about/Description.tsx

```diff
@@ -18,7 +18,7 @@ const Description = () => {
         alt="star"
         className="absolute bottom-[-50%] right-[20%] w-[3.5vw]"
       />
-      <p className="mt-[2%] w-[60%] font-hearts text-[1.8vw] font-light text-hearts-brown">
+      <p className="mt-[2%] w-[60%] font-hearts text-[3vw] font-light text-hearts-brown lg:text-[1.8vw]">
```

Comment:
check experience and todos components in about section and try to match width/text size